test/e2e/uicatalog/find-specs.js

```diff
@@ -0,0 +1,378 @@
+import env from '../helpers/env';
+import setup from "../setup-base";
+import desired from './desired';
+import B from 'bluebird';
+import _ from 'lodash';
+import { clickButton, throwMatchableError, filterDisplayed,
+         filterVisibleUiaSelector } from '../helpers/recipes';
+
+console.log(env.IOS8, env.IOS9);
```

Comment:
No. I need to clean this up. I just wanted to see how it handles in Travis (locally the whole UICatalog suite runs in 13 minutes now).